### PR TITLE
Add the possibility to remove multiple views

### PR DESCRIPTION
**Pull request by Aymenworks**

### CODE REVIEW

**Comment by muukii** [comment]
@Aymenworks 
Thank you for requesting feature.
I think added remove function should be aligned to `open func remove(view: UIView, animated: Bool)`.
For example, `open func remove(views: [UIView], animated: Bool)`.
And then in this function, using perofmBatchUpdate to keep performance.
What do you think?

**Comment by Aymenworks** [comment]
Done!

Thanks you for the feedbacks.

**Review by muukii — APPROVED** [review]
Thank you!

File: StackScrollView/StackScrollView.swift [hunk 1]
```diff
@@ -148,6 +148,50 @@ open class StackScrollView: UICollectionView, UICollectionViewDataSource, UIColl
...
+
+    // It seems that the layout is not updated properly unless the order is aligned.
+    indicesForRemove.sort(by: >)
+
+    for index in indicesForRemove {
+      self.views.remove(at: index)
+    }
+
+    if animated {
+      UIView.animate(
```
Comment (JohnEstropia) [inline StackScrollView/StackScrollView.swift:169]: I'm not sure, but doesn't `performBatchUpdates` already animate the deletion?

**Pull request merged**